cmd/syncthing/gui.go

```diff
@@ -169,6 +169,8 @@ func (s *apiSvc) Serve() {
 	postRestMux.HandleFunc("/rest/system/restart", s.postSystemRestart)        // -
 	postRestMux.HandleFunc("/rest/system/shutdown", s.postSystemShutdown)      // -
 	postRestMux.HandleFunc("/rest/system/upgrade", s.postSystemUpgrade)        // -
+	postRestMux.HandleFunc("/rest/system/stop", s.postSystemStopFolder)        // folder
+	postRestMux.HandleFunc("/rest/system/start", s.postSystemStartFolder)      // folder
```

Comment:
Debatable if this goes under /rest/system or /rest/db. We have /rest/db/scan, and we're not stopping the entire *system*, but it's not strictly speaking a db operation...